knotx-example/knotx-example-monolith/src/main/resources/knotx-example-monolith.json

```diff
@@ -12,7 +12,14 @@
         "repository.address": "template-repository",
         "engine.address": "template-engine"
       }
-    }
+    },
+    "worker" : false,
```

Comment:
remove all scaling properties from example-monotlith fro all verticles - it does not make sense here